lib/data/external/AwsClient.js

```diff
@@ -208,6 +208,22 @@ class AwsClient {
             });
         });
     }
+
+    abortMPU(key, uploadId, location, bucket, log, callback) {
+        const awsBucket = this._awsBucketName;
+        const awsKey = this._createAwsKey(bucket, key, this._bucketMatch);
+        const abortParams = {
+            Bucket: awsBucket, Key: awsKey, UploadId: uploadId,
+        };
+        return this._client.abortMultipartUpload(abortParams, err => {
+            if (err) {
+                log.trace('err from data backend on abortMPU',
+                { error: err, dataStoreNae: this._dataStoreName });
+                return callback(errors.InternalError);
```

Comment:
let's customize the error description with something like:
'There was an error aborting the MPU on AWS S3. You should abort directly on AWS S3 using the same uploadId.'

Otherwise they will have an orphan on AWS.